python/nano/test/pytorch/tests/test_inference_optimizer.py

```diff
@@ -464,3 +465,25 @@ def test_context_manager(self):
         model, option = inference_opt.get_best_model()
         with InferenceOptimizer.get_context(model):
             pass
+
+    def test_inplace(self):
+        class CannotCopyNet(Net):
+            def __deepcopy__(self, memo):
+                invalidOperationError(False, "The `deepcopy` function shouldn't be called")
+
+        model = CannotCopyNet()
+        inference_opt = InferenceOptimizer()
+        # int8
+        int8_model = inference_opt.quantize(model, calib_data=self.train_loader, inplace=True)
+        # bf16+ipex
+        bf16_ipex_model = inference_opt.quantize(model, calib_data=self.train_loader, precision='bf16', use_ipex=True, inplace=True)
+        # ipex
+        ipex_model = inference_opt.trace(model, input_sample=self.train_loader, use_ipex=True, inplace=True)
```

Comment:
could we test like this? I mean we state `inplace=True` for all cases and the model is now very messy which is not a  good choice for cascade calling of `quantize`/`trace` right?

Reply:
fix it